src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.tree');
+
+table.onclick = function (e) {
+  const target = e.target;
+
+  if (target.tagName !== 'SPAN') {
+    return;
+  }
+
+  if (target.nextElementSibling.style.display === 'none') {
```

Comment:
Ensure that the initial display state of the elements you want to toggle is set to 'none'. Otherwise, the toggling logic might not work as expected on the first click.